addons/website/views/snippets/s_discovery.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_discovery" name="Discovery">
+    <section class="s_discovery o_cc o_cc1 pt136 pb136">
+        <div class="container">
+            <span class="s_cta_badge d-inline-block my-3 border rounded py-2 px-3 o_cc o_cc1 o_animable" data-snippet="s_cta_badge" data-name="CTA Badge" style="border-radius: 32px !important; text-align: center;">
+                <i class="fa fa-1x fa-fw fa-info-circle o_not-animable" role="img"/>&#160;&#160;What do you want to promote&amp;nbsp;? &#160;&#160;&#160;&#160;<a href="#">See more <i class="fa fa-long-arrow-right" role="img"/></a>
```

Comment:
I know it doesn't seem logical, but if you want to define "1x" value in the WB by default you should remove `fa-1x` 😅 (I guess this should also be fixed in `cta_badge` 😉 )
<img width="1791" alt="Capture d’écran 2024-08-22 à 11 04 31" src="https://github.com/user-attachments/assets/603a7881-f253-4614-9c70-0ad68bceb478">
```suggestion
                <i class="fa fa-fw fa-info-circle o_not-animable" role="img"/>&#160;&#160;What do you want to promote&amp;nbsp;? &#160;&#160;&#160;&#160;<a href="#">See more <i class="fa fa-long-arrow-right" role="img"/></a>
```

Reply:
Not related to the `s_cta_badge` only but there's indeed an issue 😅 I'll fix this occurrences and will open a bugfix PR to fix this behaviour in `s_badge` and `s_cta_badge` 👍